### DISCUSSION TITLE
Utilize configuration file

### DISCUSSION THREAD

**Discussion by 004a** [sergi0g/cup#19] [Ideas]
Suggestion:  utilize a configuration file for the images and tags rather than attaching it to a socket.

Socket attachment works, yet will be limited to running containers that socket has direct access. Using a config file of some kind of structure, yaml, toml, or json, makes usage of cup more flexible.

**Comment by sergi0g** [comment]
@004a this is a feature request, so it was moved to #23. You can track it's progress there. Thanks for the idea!
